test/e2e/tests/deep-link/deep-link.spec.ts

```diff
@@ -87,7 +88,7 @@ describe('Deep Link', function () {
       'signed without sig_params',
       'unsigned',
     ] as const,
-    ['/home', '/swap', '/INVALID', REWARDS_ROUTE] as const,
+    ['/home', '/swap', '/INVALID', REWARDS_ROUTE, '/settings'] as const,
```

Comment:
you should add a new test for this, and not expand the scenarios matrix.  The `REWARDS_ROUTE` shouldn't have been added here. I'll remove it in another PR.

Reply:
draft PR here: https://github.com/MetaMask/metamask-extension/compare/fix-tests?expand=1